### PR TITLE
Modify position of tooltip for management job list

### PR DESCRIPTION
**Pull request by nixocio**
Modify position of tooltip for management job list. Also, remove
duplicated tooltip.

### CODE REVIEW

File: awx/ui/src/components/PaginatedTable/ActionItem.js [hunk 1]
```diff
@@ -2,7 +2,13 @@ import 'styled-components/macro';
 import React from 'react';
 import { Tooltip } from '@patternfly/react-core';
 
-export default function ActionItem({ column, tooltip, visible, children }) {
+export default function ActionItem({
+  column,
+  tooltip,
+  visible,
+  position,
```
Comment (kialam) [inline awx/ui/src/components/PaginatedTable/ActionItem.js]: This prop isn't being used anywhere in the code - is this change meant for future-scoped work?

**Review by kialam — CHANGES_REQUESTED** [review]
Changes to `ActionItem.js` seem extraneous.

File: awx/ui/src/components/PaginatedTable/ActionItem.js (hunk 1 above)
Comment (nixocio) [reply awx/ui/src/components/PaginatedTable/ActionItem.js]: @kialam, yeah. It would be.

But I can remove that one.

Comment (nixocio) [reply awx/ui/src/components/PaginatedTable/ActionItem.js]: @kialam, removed.

**Review by kialam — APPROVED** [review]
👍

**Review by akus062381 — APPROVED** [review]

**Pull request merged**